include/Item.hpp

```diff
@@ -4,6 +4,8 @@
 #include <fstream>
 #include <iostream>
 #include <string>
+#include <iomanip>
```

Comment:
Have you tried to run clang-format on your code?  I ask because it orders the headers alphabetically.  Not that I care about that, but I'd recommend trying to run it on your code; so in the future it won't get changed when someone else runs it and ```git blame``` should still show you as the author.

If you're using a WSL Ubuntu install:
```sudo apt install clang-format```
```make better```